### ISSUE TITLE
Error when attempting to build GCAM datasystem

### ISSUE THREAD

**Issue by tsgsteele** [JGCRI/gcamdata#1163]
I was able to successfully build the gcamdata package from gcamdata.Rproj on Windows, but the driver repeatedly crashes at the same point:

```
[1] "module_socioeconomics_L100.Population_downscale_ctry"
[1] "- make 0.63"
[1] "module_socioeconomics_L101.Population"
[1] "- make 0.25"
[1] "module_socioeconomics_L102.GDP"
Error in if (!is.data.frame(groups) || tail(names(groups), 1L) != ".rows") { : 
  missing value where TRUE/FALSE needed
```
It seems to be a problem in the chunk where the last module before the error is located. I don't know how to identify where, any help would be greatly appreciated!

I'm running gcamdata.Rproj from the master downloaded from Github. I've tried running from the cloned repo and I get the same error message

**Comment by michaeljaywalsh** [comment]
I just got the same error.

**Comment by bpbond** [comment]
Can you provide the output of `sessionInfo()` after this error? Thanks.

**Comment by tsgsteele** [comment]
Looking at past closed issues, I tried reverting to dplyr 0.8.5, this seemed to work and I can now build the datasystem fine. Thank you!

**Comment by bpbond** [comment]
That's what it sounded like; great.
